deploy_from_github_zip.bash

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+[ $# -eq 0 -o $# -gt 3 ] && {
+    echo "Usage: deploy_from_github_zip.bash <new project name> [GitHub-user/repository-name] [branch]"
+    exit 1
+}
+
+project=$1
+repo=${2-LSSTDESC/start_paper}
+branch=${3-master}
```

Comment:
It looks like you've deleted this script from the `doc` branch, which is where I would have expected it to live from now on, given your comments. Where would you like me to make any changes to it?